randovania/data/json_data/prime3.txt

```diff
@@ -1214,25 +1386,36 @@ Asset id: 6613311822671481119
   > Pickup (Missile Expansion)
       Trivial
 
+> Event - Leviathan Shield; Heals? False
+  * Event Jungle Leviathan Shield Generator
+
 ----------------
 Hidden Court
 Asset id: 14006603351279167667
 > Dock to Hidden Court Hall; Heals? False
   * Normal Door to Hidden Court Hall/Dock to Hidden Court
+  > Dock to Burrow
+      All of the following:
+          After Hidden Court Burrow Gate
+          Grapple Swing or Use Screw Attack (Space Jump)
+  > Bottom Floor
+      Trivial
   > Bottom of Halfpipe
       Trivial
 
 > Dock to Ruined Shrine; Heals? False
   * Normal Door to Ruined Shrine/Dock to Hidden Court
   > Bottom Floor
-      Trivial
+      After Hidden Court Golem
 
 > Dock to Burrow; Heals? False
   * Normal Door to Burrow/Dock to Hidden Court
-  > Bottom Floor
+  > Dock to Hidden Court Hall
+      All of the following:
+          After Hidden Court Burrow Gate
+          Grapple Swing or Use Screw Attack (Space Jump)
+  > Event - Burrow Gate
       Trivial
-  > Bottom of Halfpipe
-      Grapple Swing or Use Screw Attack (Space Jump)
```

Comment:
This dock should connect to the gate event (already done), and Bottom Floor with the requirement "After Gate". Remove connection to Hidden Court Hall.